src/components/ContactSection/ContactSection.js

```diff
@@ -5,11 +5,11 @@ import { LinkedinSquare } from '@styled-icons/boxicons-logos/LinkedinSquare';
 import { Envelope } from '@styled-icons/boxicons-regular/Envelope';
 import { Phone } from '@styled-icons/boxicons-regular/Phone';
 
-import { colors, breakPoints } from 'styles/base';
-import SectionWrapper from 'styles/elements/SectionWrapper';
-import SectionHeader from 'styles/elements/SectionHeader';
-import SectionItemWrapper from 'styles/elements/SectionItemWrapper';
-import SectionItemIcon from 'styles/elements/SectionItemIcon';
+import { breakPoints } from 'styles/base';
+import SectionWrapper from '../SectionWrapper';
```

Comment:
Let's create styled file, export all those styled component by name. It doesn't have sense to keep separate file for each styled components